### DISCUSSION TITLE
Filtering and sorting the project list by a number of things

### DISCUSSION THREAD

**Discussion by czerwinskie** [Simon-Initiative/oli-torus#3215] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is difficult to manage/administer and find projects and troubleshoot/support without some sort of filtering/sorting of projects.  

Even for folks who are collaborators on more than one project, it is difficult to find exactly what is needed. It would be nice to be able to determine what my course list shows as default.

Ideas for filtering/sorting options: Institution, author, status, collaborator names, dates, "lineage/source" (all projects copied from existing), keyword in title and descriptions.

[possibly need an exercise where we show the tech team where/how things exist now to get ideas for all the admin tools needed]

**Comment by ThinkThoughtThunk** [comment]
"courses I'm in" - some way to see your courses vs your "side" (Added to) courses?

**Comment by ThinkThoughtThunk** [comment]
Possible UX discussion topic

**Comment by czerwinskie** [comment]
The LE team discussed this request and came up with a number of things needed to find and help with course administration. Not sure where to put them all, so I'll put them here. These are ideas for what we should collect/find about each project. These things will also help with understanding intent when these projects are used for research/data.


Search by any/all collaborators
Source material/parent project/cloned from
Has published things from it - projects and sections 

Some idea of a size: 
- Pages
- sidecar/semester/etc.
- # question/activities
- YouTube references
- Superactivites

How many sections
Total number of enrollments (across institutions/products/etc.)
Primary institution (is it linked to person or institution)
Datashop repositories
Products 
License information
Marketing language (similar to a description but can be used for outreach)
Primary domain/field

Need to figure out the best “place” for each
Need advanced search/queries 
Required fields in the description, etc.
